CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - It's safe now to use `ChatChannel` and `ChatMessage` across multiple threads [#984](https://github.com/GetStream/stream-chat-swift/pull/984)
 - Web socket reconnection logic better handles the "no internet" errors [#970](https://github.com/GetStream/stream-chat-swift/pull/970)
 
+### ✅ Added
+- Expose the entire quoted message on `ChatMessage` instead of it's `id` [#992](https://github.com/GetStream/stream-chat-swift/pull/992)
+
+### ⛔️ Deprecated
+- `ChatMessage`'s property `quotedMessageId` is now deprecated. Use `quotedMessage.id` instead [#992](https://github.com/GetStream/stream-chat-swift/pull/992)
```

Comment:
nit: You can just write `ChatMessage.quotedMessageId` is .... it's easier to read :) Also, `quotedMessage.id` doesn't compile, it must be `quotedMessage?.id`

Reply:
done ✅